src/app/state/account/selectors.test.ts

```diff
@@ -0,0 +1,171 @@
+import { TRANSACTIONS_LIMIT } from '../../../config'
+import { RootState } from '../../../types'
+import { Transaction, TransactionStatus } from '../transaction/types'
+import { hasAccountUnknownPendingTransactions } from './selectors'
+
+const currentAddress = 'oasis1qz78ap0456g2rk7j6rmtvasc9v2kjhz2s58qgj90'
+
+describe('hasAccountUnknownPendingTransactions', () => {
```

Comment:
Some tests seem like duplication of `src/app/pages/AccountPage/Features/TransactionHistory/__tests__/index.test.tsx`, but since they are unit tests, looks fine.